execution_introspection_test.go

```diff
@@ -53,6 +53,9 @@ func TestIntrospectionQuery(t *testing.T) {
 
 	input CinemaInput {
 		name: String! = "SomeName"
+		cinema: CinemaInput = {name: "SomeOtherName"}
```

Comment:
is this a valid input type? Would it create a infinite chain of `cinema` values.

Reply:
Yes indeed:
https://spec.graphql.org/October2021/#sec-Input-Objects.Circular-References